### PR TITLE
Relative path configurable

### PR DESCRIPTION
**Pull request by ahgittin**
extends #160 offering a way for downstream projects to customize relative path if it causes issues

### CODE REVIEW

**Comment by ahgittin** [comment]
thanks @tbouron for the suggestion in #160 , implemented here

File: pom.xml [hunk 1]
```diff
@@ -85,6 +84,8 @@
         <brooklyn.version>1.0.0-SNAPSHOT</brooklyn.version><!-- BROOKLYN_VERSION -->
         <build.version>${revision}</build.version>
         <build.name>Apache Brooklyn</build.name>
+        <brooklyn.ui.relativePath.to.brooklyn.server.parent>../brooklyn-server/parent/</brooklyn.ui.relativePath.to.brooklyn.server.parent>
```
Comment (tbouron) [inline pom.xml]: The name is a bit too much for my taste @ahgittin, I would use a simpler `parent.relative.path`.
But happy to go ahead and merge if it works

Comment (ahgittin) [reply pom.xml]: assuming this is set higher up in a complex project hierarchy it needs to be namespaced appropriately.  someone else might have a different `parent.relative.path`.  agree it's a little bit silly long but at least it's totally unambiguous. :)

**Comment by ahgittin** [comment]
Seems maven doesn't actually respect a variable in relative path.  I must have been taking some cached or downloaded item when i tested, as when we change the version it breaks.

Also irritating, seems maven won't look in the local reactor model so we need either relative path to work or the upstream item to have been built.

Revert this.

**Pull request merged**